src/main/resources/org/opentripplanner/apis/gtfs/schema.graphqls

```diff
@@ -3148,6 +3169,8 @@ type QueryType {
         "Preferences for vehicle parking"
         parking: VehicleParkingInput
 
+        mobilityProfile: String
```

Comment:
You're not using the `MobilityProfile` enum that you defined above.

If you want to use the full power of the type system, including validation, you can do that. This means that you have to write a mapper [like this](https://github.com/opentripplanner/OpenTripPlanner/blob/6de1cf2e98b32a278aef805bb1f8a89ed0cbd398/src/main/java/org/opentripplanner/apis/gtfs/mapping/AlertEffectMapper.java#L3) but in the other direction.

Reply:
If you want to do that, you also have to regenerate the Java files from the schema.

To do that go to `src/main/java/org/opentripplanner/apis/gtfs/generated/` and then run `npm install` and `npm run generate`.

Reply:
Per our discussion, I will remove the enum in the GraphQL schema, so the can simply pass the mobility profile as free text including symbols.

Reply:
Removed in https://github.com/ibi-group/OpenTripPlanner/pull/197/commits/8c4e90b9792d8b8d3063b79b80b75e10991841ce.